### PR TITLE
320 Fade music volume between level transitions

### PR DESCRIPTION
**Pull request by smeas**

### CODE REVIEW

File: Assets/Scripts/Audio/LevelMusicPlayer.cs [hunk 1]
```diff
@@ -45,22 +47,24 @@ public class LevelMusicPlayer : SingletonBehaviour<LevelMusicPlayer> {
...
-			Stop();
+	private AudioClip GetTrackForLevel(int levelIndex) {
+		if (levelIndex == -1 || levelIndex >= tracks.Length)
+			return null;
+
+		return tracks[levelIndex];
 	}
 
-	private void Play(AudioClip track) {
+	private void Play(AudioClip track, bool allowFade) {
```
Comment (MrJackey) [inline Assets/Scripts/Audio/LevelMusicPlayer.cs:67]: Where do you use allowFade? Can't find any.

**Review by MrJackey — CHANGES_REQUESTED** [review]
Tiny possible unnecessary picky thing. Otherwise seems to work well

**Review by Addehman — APPROVED** [review]
Noice!

**Review by MrJackey — APPROVED** [review]
:shipit:

**Pull request merged**